selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/PageObjectsInjectorImpl.java

```diff
@@ -32,12 +39,22 @@
 
   @Override
   public Map<Class<?>, Object> getDependenciesWithWebdriver(SeleniumWebDriver seleniumWebDriver) {
+    SeleniumWebDriverHelper seleniumWebDriverHelper =
+        new SeleniumWebDriverHelper(
+            seleniumWebDriver, new WebDriverWaitFactory(seleniumWebDriver), getActionFactory());
+
     Map<Class<?>, Object> dependencies = new HashMap<>();
     dependencies.put(
         Entrance.class,
         cheSeleniumWebDriverRelatedModule.getEntrance(
-            isMultiuser, new CheLoginPage(seleniumWebDriver), seleniumWebDriver));
+            isMultiuser,
+            new CheLoginPage(seleniumWebDriver, seleniumWebDriverHelper),
+            seleniumWebDriver));
 
     return dependencies;
   }
+
+  private ActionsFactory getActionFactory() {
```

Comment:
It looks like a duplication of "CheSeleniumSuiteModule#getActionFactory()".
Please, try to inject "ActionFactory" to current class:
```
  @Inject private ActionsFactory actionsFactory;
```

Reply:
Good catch! Will check you suggestion.